flavors/standard-EXASOL-8.0.0/flavor_base/flavor_base_deps_python/packages/python3_pip_packages

```diff
@@ -0,0 +1,66 @@
+azure-batch|11.0.0
+azure-cosmos|4.2.0
+azure-eventgrid|4.7.1
+azure-eventhub|5.7.0
+azure-eventhub-checkpointstoreblob|1.1.4
+azure-eventhub-checkpointstoreblob-aio|1.1.4
+azure-identity|1.6.1 # Max version supported by azure-kusto-data
+azure-keyvault|4.1.0
+azure-keyvault-certificates|4.3.0
+azure-keyvault-keys|4.4.0
+azure-keyvault-secrets|4.3.0
+azure-kusto-data|2.3.2
+azure-loganalytics|0.1.1
+azure-servicebus|7.5.0
+azure-storage-blob|12.9.0
+azure-storage-file-datalake|12.5.0
+azure-storage-file-share|12.6.0
+azure-storage-queue|12.1.6
+boto3|1.20.37
+google-cloud-asset|3.7.1
+google-cloud-bigquery|2.32.0
+google-cloud-bigquery-storage|2.11.0
+google-cloud-bigtable|2.4.0
+google-cloud-containeranalysis|2.6.3
+google-cloud-datacatalog|3.6.2
+google-cloud-datastore|1.15.3 # Max version supported by google-cloud-ndb
+google-cloud-firestore|2.3.4
+google-cloud-kms|2.10.1
+google-cloud-logging|2.7.0
+google-cloud-monitoring|2.8.0
+google-cloud-ndb|1.11.1
+google-cloud-pubsub|2.9.0
+google-cloud-spanner|3.12.1
+google-cloud-storage|2.0.0
+google-cloud-trace|1.5.1
+martian|1.4
+protobuf|3.19.3
+pyexasol|0.23.3
+pykickstart|3.35
```

Comment:
Do we really need to be able to parse kickstart files, which are unattended install files for RedHat